docs/test_plan/proctored_exam.rst

```diff
@@ -0,0 +1,126 @@
+#. Clicking the "Review Required" button spawns a modal that says "Update review status"
+#. Clicking "Cancel" closes modal
+#. Clicking "Verify" marks attempt as verified both in the dashboard
+#. Check the Gradebook (Instructor Dashboard -> Student Admin -> Gradebook) and check that the grade for the exam is 100.
+#. Clicking "Reject" marks attempt as reject both in the dashboard
+#. Check the Gradebook (Instructor Dashboard -> Student Admin -> Gradebook) and check that the grade for the exam is 0.
+#. Clicking the "Review Dashboard" link in the modal "loads" the review dashboard (NOTE: This shouldn't work yet, and will load a broken page instead)
+
+Reviewing Verified Attempts
+===========================
+#. Exam attempts with a status of "Verified" have a button/link at the end of their rows titled "Manual Review"
+#. A modal pops up upon clicking "Manual Review" that contains "Cancel" and "Reject" Buttons
+#. Clicking "Cancel" closes the modal
+#. Clicking "Reject" changes the attempt status to "rejected" in the UI
+#. Upon refreshing a filtering for this exam attempt, it is still marked as "rejected"
+#. Check the Gradebook (Instructor Dashboard -> Student Admin -> Gradebook) and check that the grade for the exam is 0.
+#. Check the Generated Certificates table in the LMS admin at "{your platform url}/admin/certificates/generatedcertificate/" to see if the user's certificate for the course has been marked as "invalidated"
+
+Reviewing Rejected Attempts
+===========================
+#. The same exam attempts with a status of "Rejected" have a button at the end of their rows saying "Manual Review"
+#. A modal pops up upon clickin "Manual Review" that only has the "Cancel" and "Verify" Buttons
+#. Clicking "Cancel" closes the modal
+#. Clicking "Verify" changes the attempt status to "verified" in the UI
+#. Upon refreshing a filtering for this exam attempt, it is still marked as "verified"
+#. Check the Gradebook (Instructor Dashboard -> Student Admin -> Gradebook) and check that the grade for the exam is 100.
+
+Reviewing "Errored" attempts
+============================
+Instructions: Create a couple of errored attempts by opening Chrome DevTools during an exam
+#. Clicking the "Review Required" button pops up a modal that says "Update review status"
+#. Clicking "Cancel" closes modal
+#. Clicking "Verify" marks attempt as verified both in the dashboard and in the edx-exams admin
+#. Clicking "Reject" marks attempt as reject both in the dashboard and in the edx-exams admin
+
+Resetting an exam attempt
+=========================
+#. Clicking "Reset" on an exam attempt brings up a confirmation modal
+#. Clicking "No (Cancel)" closes the modal and nothing changes
+#. Clicking "Yes I'm Sure" removes the exam attempt from the table (effectively confirming that it's been reset)
+#. The exam attempt no longer appears in the edx-exams admin
```

Comment:
NIT: this one part of the test requires additional access. Going back the the section as a learner and ensuring you get the correct interstitial can test the reset happened without needing admin. (the tester already has to login as a learner to check completion in the next step.